scripts/uninstall.sh

```
@@ -23,46 +23,46 @@ if [ "$(id -u)" -ne 0 ]; then
+    $SUDO systemctl disable llmsnap
 }
 if available systemctl; then
     configure_systemd
 fi
 
-if available llama-swap; then
-    status "Removing llama-swap binary..."
-    $SUDO rm $(which llama-swap)
+if available llmsnap; then
+    status "Removing llmsnap binary..."
+    $SUDO rm $(which llmsnap)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)